pan3d/explorers/globe.py

```diff
@@ -300,34 +297,33 @@ def _on_color_by(self, color_by, **__):
             self.state.color_min = 0
             self.state.color_max = 1
 
-    @change("color_preset", "color_min", "color_max", "nan_color")
+    @change("color_preset", "color_min", "color_max", "nan_color", "opacity")
```

Comment:
opacity could have been in its own callback as there is no relationship with color and range.